crates/matrix-sdk-base/src/client.rs

```diff
@@ -134,14 +138,23 @@ impl fmt::Debug for BaseClient {
     }
 }
 
+/// Whether this client instance supports threading or not
```

Comment:
We should explain a bit more what this entails: in-thread events might be excluded from timeline by default, unread counts will ignore threads, and so on.